courses/templates/courses/introduction-to-data-mining.html

```diff
@@ -0,0 +1,148 @@
+    <p style="padding-left:30px;"><b>Classification models</b>. Classification trees, forests, logistic regression,
+        nearest neighbors classification.
+        Data painting. Area under ROC curve. Analysis of results of classifications. Nomograms.
+    </p>
+
+    <h5>September 15, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Regression</b>. Linear regression, polynomial regression and overfitting.
+        Regularization.
+        Effects of regularization on training and test sets.
+        Random forests for regression.</p>
+
+    <h5>September 19, 2015. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Partitional clustering</b>. Introduction to k-means clustering.
+        Differences with hierarchical clustering. Silhouette scoring of clustering results. Outlier detection.</p>
+
+    <h5>September 20, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Data projections</b>. Principal component analysis. Multi-dimensional scaling
+        and t-SNE.</p>
+
+    <h5>September 22, 2017. 10:00-12:00. N315</h5>
+    <p style="padding-left:30px;"><b>Gentle introduction to deep learning</b>. Image embedding. Image data analysis
+        – examples with clustering,
+        classification and outlier detection with images. Course wrap-up.</p>
+</div>
+<br>
+
+<div class="panel panel-warning">
+    <div class="panel-heading">
+        Lecturers
+    </div>
+
+    <div class="panel-body">
+        <div class="wrapper">
+            <div class="one" style="padding-top:1em">
+                <img style="float: left;"  src="{% static 'courses/janez_demsar_bw3.png' %}" alt="" />
+            </div>
+            <div class="two">
+                <blockquote style="overflow:hidden;">
+                    <p>»Playing with data is fun. It's like a detective story, where data gives you clues and you dig ever deeper into the mystery until finding the hidden treasure, the cunning murderer, or the mischievous gene.«</p>
+                    <footer>Janez Demšar</footer>
+                </blockquote>
+            </div>
+        </div>
+
+
+        <div class=wrapper" style="padding-top:1em">
+            <div class="one">
+                <img style="float: left;"  src="{% static 'courses/blaz_zupan_bw_light2.png' %}" alt="" />
+            </div>
+            <div class="two">
+                <blockquote style="overflow:hidden;"><p>"Lately, I am fascinated by how easy it is to explain machine learning to anybody. In just a few hours. To the point where a course attendee would go home and use machine learning on her own data. Or on any data from the web. And she would understand how artificial intelligence helps Google, Amazon, and Netflix. For such training, you need the right tools. And it is here where Orange shines. " </p>
+                    <footer>Blaž Zupan</footer>
+                </blockquote>
+            </div>
+        </div>
+    </div>
+
+</div>
```

Comment:
I suggest some extra space after panel-warning. Title is too close to panel for my opinion.